amp/repository/aim/view/viewActivityDifferences.jsp

```diff
@@ -45,8 +45,17 @@
 	<html:hidden property="showMergeColumn" styleId="showMergeColumn"/>
 	<html:hidden property="method" styleId="method"/>

```

Comment:
lets add a line break in the view so it is clear its a new activity

Reply:
I pasted a screenshot where the linebreak should go in order to give a bit of space between activities
![image](https://user-images.githubusercontent.com/15792006/53881987-d5ea0f80-3ff3-11e9-8662-974fbbd3bcc9.png)